### ISSUE TITLE
tokenizer crashing on bad input in 0.13

### ISSUE THREAD

**Issue by bbarker** [purescript/purescript#3694]
If I switch out the valid line above with something that doesn't make sense:

```purescript
      idNew <- pure {id: "FooBar", idType: MJ.PURL} -- Good
      idNew <- {"FooBar"} -- Bad
```

I get this compiler error:

```
brandon@af88f6acf980:/wd$ npm run testbrowser

> purescript-metajelo@ testbrowser /wd
> ./scripts/testbrowser

* Building project in /wd
purs: Internal parser error: Invalid name: SourceToken {tokAnn = TokenAnn {tokRange = SourceRange {srcStart = SourcePos {srcLine = 149, srcColumn = 17}, srcEnd = SourcePos {srcLine = 149, srcColumn = 25}}, tokLeadingComments = [], tokTrailingComments = []}, tokValue = TokString "FooBar" "FooBar"}
CallStack (from HasCallStack):
  error, called at src/Language/PureScript/CST/Utils.hs:79:17 in purescript-0.13.0-2mLnpIbmSLE8KJ0K8giZUZ:Language.PureScript.CST.Utils
purs: thread blocked indefinitely in an MVar operation

* ERROR: Subcommand terminated with exit code 1
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! purescript-metajelo@ testbrowser: `./scripts/testbrowser`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the purescript-metajelo@ testbrowser script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/brandon/.npm/_logs/2019-07-02T17_49_41_727Z-debug.log
```

Not a huge issue, as this was obviously bad code and easy to figure out from the parser crash.

**Comment by natefaubion** [comment]
Duplicate of #3689. This will be fixed in 0.13.1.

**Comment by hdgarrood** [comment]
This is a duplicate of #3689. Thanks for the report though!
